src/io/flutter/ProjectOpenActivity.java

```diff
@@ -71,6 +76,21 @@ public void runActivity(@NotNull Project project) {
     // TODO(messick) Re-enable this after dropping support for 2020.2.
     //excludeAndroidFrameworkDetector(project);
 
+    // Report time when indexing finishes.
+    DumbService.getInstance(project).runWhenSmart(() -> {
+      if (!indexingFinished) {
```

Comment:
Is this method ever actually called with (indexingFinished == true)?
If you are trying to catch an internal logic error, change this to 
assert(!indexingFinished) to clarify the intent.

Reply:
Oh right. I don't need this, was thinking this might run multiple times but that's not the case.